bin/bot.js

```diff
@@ -1,21 +1,24 @@
 import { setTimeout } from 'node:timers/promises'
 import { ethers } from 'ethers'
 import {
-  pandoraServiceAbi,
+  filecoinWarmStorageServiceAbi,
+  serviceProviderRegistryAbi,
   pdpVerifierAbi,
   sampleRetrieval,
-  testLatestRetrievableRoot,
+  testLatestRetrievablePiece,
 } from '../index.js'
 
 const {
   FLY_REGION,
   GLIF_TOKEN,
   RPC_URL = 'https://api.calibration.node.glif.io/',
-  PDP_VERIFIER_ADDRESS = '0x5A23b7df87f59A291C26A2A1d684AD03Ce9B68DC',
-  PANDORA_SERVICE_ADDRESS = '0xf49ba5eaCdFD5EE3744efEdf413791935FE4D4c5',
+  PDP_VERIFIER_PROXY_ADDRESS = '0xf9f521c6e11A1680ead3eDD8a2757Ea731458617',
+  FILECOIN_WARM_STORAGE_SERVICE_PROXY_ADDRESS = '0xfa564144f183E4E7B8FEdCfbAa412afc83D5aE3d',
+  // TODO: replace with the actual address
+  SERVICE_PROVIDER_REGISTRY_PROXY_ADDRESS = '0x',
   CDN_HOSTNAME = 'calibration.filcdn.io',
   DELAY = 1_000,
-  FROM_PROOFSET_ID = 200,
+  FROM_DATASET_ID = 0,
```

Comment:
```suggestion
  FROM_DATA_SET_ID = 0,
```

We are consistently calling it "data set" now